scripts/spack/packages/spheral/package.py

```diff
@@ -70,6 +70,7 @@ class Spheral(CachedCMakePackage, CudaPackage, PythonPackage):
     depends_on('py-sphinx@1.8.5', type='build', when='+docs')
     depends_on('py-sphinx-rtd-theme@0.5.0', type='build', when='+docs')
 
+    depends_on('caliper', type='build')
```

Comment:
We should be able to use the static version of this library, try adding `~shared` to this spec. Then we can remove caliper from our rpaths and import `libcaliper.a`.